src/main/java/org/junitpioneer/jupiter/params/RangeSourceArgumentsProvider.java

```diff
@@ -63,17 +78,16 @@
 			throw new IllegalArgumentException(message);
```

Comment:
This message may be more helpful if we explain why, even though the annotation is repeatable, it can only occur once on `@ParameterizedTests`.